lib/get-config.js

```diff
@@ -19,7 +22,11 @@ const ALLOWED_ERROR_CODES = [
 ];
 
 function resolveConfigPath(configPath, pluginPath) {
-  let usedPath = path.resolve(process.cwd(), configPath || '.template-lintrc');
+  let usedPath = path.resolve(process.cwd(), configPath || CONFIG_FILE_NAME);
```

Comment:
I think this still needs some changes, I'd expect it to look something like:

```js
let usedPath = configPath ? configPath : findUp.sync(CONFIG_FILE_NAME);
```

Basically, if a `configPath` was provided, use it verbatim; otherwise use findUp.

Reply:
![image](https://user-images.githubusercontent.com/1360552/75093904-2a84f280-5597-11ea-8d76-31b10c8d0c4e.png)

for such change we have alot of issues

Reply:
@rwjblue we have to rewrite few tests to get new logic support and be sure it won't break any existing apps. current changes lands new feature "gracefully" (without regression) and if we trying to fix our initial logic new "feature" break alot of tests.

Is it possible to land "enhansment" and create new issue about clear resolution logic?
(or I'm just  panicking or doins something worng)

Reply:
@rwjblue 
failing tests:
*  uses an empty set of rules if no .template-lintrc is present
* should be able to instantiate without options
* accepts a fake console implementation

![image](https://user-images.githubusercontent.com/1360552/75094382-85b8e400-559b-11ea-83e3-1edc1d2b7f84.png)

Reply:
looks like I found valid fix

```js
  try {
    if (!filePath) {
      // tests:
      // uses an empty set of rules if no .template-lintrc is present
      // should be able to instantiate without options
      return {};
    }
    require.resolve(filePath);
  } catch (e) {
```

Reply:
Thanks for pushing up the change, I'll take a look also (maybe I'm misunderstanding something)

Reply:
The logic sharing makes this a bit annoying, lemme do a small cleanup then I'll merge (I'll push to this branch directly)